packages/prices/utils/price.ts

```diff
@@ -113,6 +115,50 @@ export const getCurrency = (
 	};
 };
 
+/**
+ * Get the integer value from the decimal price.
+ *
+ * @param {number} priceInt The price in decimals.
+ * @param {string} thousandSeparator The thousand separator.
+ * @param {number} minorUnit The number of decimals to display.
+ * @return {string} The extracted integer value.
+ */
+export const getIntegerValue = (
+	priceInt: number,
+	thousandSeparator: string,
+	minorUnit: number
+): string => {
+	return Math.floor( priceInt / 10 ** minorUnit )
+		.toString()
+		.replace( /\B(?=(\d{3})+(?!\d))/g, thousandSeparator );
+};
+
+/**
+ * Get the decimal value from the decimal price.
+ *
+ * @param {number} priceInt The price in decimals.
```

Comment:
What does "The price in decimals" mean? It is unclear to me what's being passed in.

Reply:
See https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5125#discussion_r754783144